File: Nikita/GFG-Arrays-main/Easy/level 1/Reverse a String.py
#User function Template for python3

def reverseWord(s):
    s1=s[::-1]
    return s1
    #your code here

#{ 
#  Driver Code Starts
#Initial Template for Python 3

if __name__ == "__main__":
    t = int(input())
    while(t>0):
        s = input()
        print(reverseWord(s))
        t = t-1

# } Driver Code Ends

File: JyotiKumari/Array(GFG)/Reverse a string.cpp
void swap(string a, string b){
    string temp;
    temp = a;
    a= b;
    b = temp;
}

string reverseWord(string str){
    
  //Your code here
  int len = str.size();
   for (int i = 0; i < len/2; i++)  
    {  
        // temp variable use to temporary hold the string  
        swap(str[i],str[len-i-1]);
    }  
  return str;
}